packages/maskbook/src/plugins/Wallet/SNSAdaptor/WalletStatusDialog/RecentTransactionList.tsx

```diff
@@ -47,6 +42,11 @@ export function RecentTransactionList(props: RecentTransactionListProps) {
     const chainId = useChainId()
     const { value: transactions, error, retry } = useRecentTransactions()
 
+    console.log('DEBUG: recent transaction')
+    console.log({
+        transactions,
+    })
+
```

Comment:
Do you forget to remove this debug code?

Reply:
DONE